tests/modules/sms_test_gateway/src/Plugin/SmsGateway/DefaultCapabilities.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\sms_test_gateway\Plugin\SmsGateway;
+
+use Drupal\sms\Plugin\SmsGatewayPluginBase;
+use Drupal\sms\Message\SmsMessageInterface;
+
+/**
+ * Defines a gateway for testing default capabilities defined by annotation.
```

Comment:
I was a bit confused about this gateway at first until I read through the tests. However, I think if we tweaked this description a bit to `* Defines a gateway for testing default values of capabilities defined by annotation.`, it would have been more helpful. If course, fitting that into 80 characters is the big challenge.